src/js/containers/generateFiles/GenerateFilesContainer.jsx

```diff
@@ -60,15 +62,30 @@ class GenerateFilesContainer extends React.Component {
 			},
 			d1Status: "waiting",
 			d2Status: "waiting",
-			errorDetails: ""
+			errorDetails: "",
+			gtas: null
 		};
 	}
 
 	componentDidMount() {
 		this.isUnmounted = false;
 		this.checkForPreviousFiles();
+		this.isGtas();
 	}
 
+	isGtas() {
+		ReviewHelper.isGtasWindow()
+			.then((res) => {
+				if(res.open != this.state.gtas) {
```

Comment:
Even in the previous setup, res.open would NEVER equal this.state.gtas. Given the suggestion below and the changes to what we return from the backend, this check could be removed completely and you could always just set state.